test/iso/param-component.js

```diff
@@ -0,0 +1,173 @@
+import sinon from 'sinon';
```

Comment:
So this is the first thing in this "iso" dir, but I don't see any changes that would make CI run this in both browser and server env - what's the deal?

Reply:
the `browser` and `server` dir all import this file. 

https://github.com/mixpanel/panel/blob/hans-panel-params/test/browser/param-component.js
https://github.com/mixpanel/panel/blob/hans-panel-params/test/server/param-component.js

the original testcases are quite different in `browser` and `server` folder. but this testcase should be identical for both browser-test and server-test

Reply:
that's an interesting approach, wouldn't it make more sense for the test commands to look in the `iso` dir rather than relying on files in both browser/ and server/ dirs to import from iso?

Reply:
hmm, the `mocha` and `karma` commands target `browser` and `server` folder respectfully. 

mocha scans through the whole dir (test/server) whereas the karma relies on webpack build that has single entrypoint in `browser/index.js`

i think this is the minimum code to make it work on both test env

Reply:
Whatever, it's not worth bikeshedding more, I meant adding `test/iso` to the `test-server` command in package.json and I guess adding to the webpack build for browser tests, feels like the right spots to configure this, but not a big deal either way